packages/dev-server-storybook/package.json

```diff
@@ -58,11 +57,10 @@
   "dependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
-    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/mdx": "^2.3.0",
```

Comment:
mdx 2 is a breaking change that not directly related to this update
is it strictly necessary?

Reply:
Nope, it wasn't strictly necessary. Figured it would be good to fold into another breaking change.

Reply:
I think we should have this in Storybook 7 only
but for Storybook 6 users I'd stick to MDX 1
MDX 2 is pretty big in terms of breaking changes, I don't see how users of our current Storybook 6 prebuild setup will benefit from this